pyseq2/fluidics/valve.py

```diff
@@ -107,8 +112,8 @@ async def pos(self) -> int:
             return p1
 
         elif CONFIG.machine == "HiSeq2500":
-            if p1 == 6:
-                return 0
```

Comment:
Do you want to get rid of the safety ports or are there better ports that are unused.